src/install/unix.rst

```diff
@@ -23,6 +23,10 @@ release are the canonical sources of installation information. However, many
 systems have gotchas that you need to be aware of. In addition, dependencies
 frequently change as distributions update their archives.
 
+**Hint**: It is worth considering using packaging tools such as LXC, Docker
```

Comment:
-1 on this hint. This file is how to install on unix, so unless there will be detailed instructions on how to do that with LXC/Docker/et al this paragraph is not helping our users.